charts/smarter-device-manager/README.md

```diff
@@ -0,0 +1,91 @@
+# smarter device manager
+
+This is a helm chart for [smarter-device-manager](https://gitlab.com/arm-research/smarter/smarter-device-manager/) 
+([see also](https://community.arm.com/developer/research/b/articles/posts/a-smarter-device-manager-for-kubernetes-on-the-edge)).
+
+This tool discovers host hardware resources and manages their provisioning for the pods, decoupling host paths from them, and ensuring that pods don't compete for the same piece of hardware. 
+
+Typical use cases are Zigbee/Zwave USB dongles for other charts available in this repository (Zigbee2MQTT, ZwaveJS2MQTT).
+
+**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/k8s-at-home/charts/issues/new/choose)**
+
+## TL;DR;
+
+```shell
+$ helm repo add k8s-at-home https://k8s-at-home.com/charts/
+$ helm install k8s-at-home/smarter-device-manager
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release k8s-at-home/smarter-device-manager
+```
+
+Please consider overrading the default configuration through `values.yaml` file. 
+For example: 
+```yaml
+config:
+  - devicematch: ^snd$
+    nummaxdevices: 20
+  - devicematch: ^rtc0$
+    nummaxdevices: 20
+  - devicematch: ^video[0-9]*$
+    nummaxdevices: 20
+  - devicematch: ^ttyACM[0-9]*$
+    nummaxdevices: 1
+```
+The above configuration allows up to twenty pods to make use of `/dev/snd`, `/dev/rtc0` and each of `/dev/video[0-9]*` devices. 
+Only one pod at a time can use any of the discovered host `/dev/ttyACM[0-9]*` devices. This means that if a pod has reserved a dongle at `/dev/ttyACM0`, another pod requesting the same device will stay pending.
+
+Please note that only the root of the host `/dev` directory is considered for discovery. Therefore, `by-id` paths will not work and some `udev` rules may be necessary.
+
+The hardware are requested by pods through `resources`, e.g.:
+```yaml
+  resources:
+    limits:
+      smarter-devices/ttyUSB-Conbee-2: 1
+    requests:
+      smarter-devices/ttyUSB-Conbee-2: 1
+```
+
+Common considerations about node affinity apply. 
```

Comment:
What do you mean with this exactly? I thought that when you use this solution it is no longer required to also use node affinity?

Reply:
Well, it means that the thing effectively works like affinity. Which is very confusing. Removed.